homeassistant/components/lutron/fan.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+from typing import Any
+import logging
+
+from homeassistant.components.fan import FanEntity, FanEntityFeature
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
+
+from . import LUTRON_CONTROLLER, LUTRON_DEVICES, LutronDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+def setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Set up the Lutron fans."""
+    devs = []
+    for area_name, device in hass.data[LUTRON_DEVICES]["fan"]:
+        dev = LutronFan(
+            area_name, 
+            device, 
+            hass.data[LUTRON_CONTROLLER],
+            device.uuid, 
+            device.name, 
+            None)
+        devs.append(dev)
+
+    add_entities(devs, True)
+
+
+class LutronFan(LutronDevice, FanEntity):
+    _attr_should_poll = False
+
```

Comment:
```suggestion
    _attr_should_poll = False
    _attr_supported_features = FanEntityFeature.SET_SPEED
    _attr_speed_count = 4

```

Reply:
@gjohansson-ST I included this based on it being in the fan template.  My assumption was this was a coding standard.  I'm open to removing if that is not the case.
ha-core/homeassistant/components/template/fan.py

Reply:
For static assignment we typically want to see it as class attributes